.env

```diff
@@ -41,8 +41,10 @@ DJANGO_LOGGING_FORMATTER=human_readable
 # Applies to both Django and Celery Workers
 DJANGO_LOGGING_LEVEL=INFO
 DJANGO_WORKERS=4 # Defaults to the maximum available based on CPU cores if not set.
```

Comment:
What about this one?

We should review the whole `.env` file.

Reply:
Done!